### ISSUE TITLE
SOS VerifyHeap and Dump

### ISSUE THREAD

**Issue by davidwrighton** [dotnet/runtime#1599]

**Comment by davidwrighton** [comment]
This issue is impeding analysis of issues on the 3.1 runtime.

**Comment by mikem8361** [comment]
David, could I get a little more detail on what has changed in the runtime and what needs to change in the DAC.  Is this the GC change to 6 generations?

**Comment by davidwrighton** [comment]
No, this appears to be a long-standing issue that has been in the product for years. The issue is that the global allocation context isn't typically used in situations where verify heap is used, and thus we didn't find the problem earlier. To reproduce the issue you must configure the gc to expect there to be only 1 processor, and thus skip usage of the thread specific allocation context.
